docs/sources/reference/components/mimir/mimir.alerts.kubernetes.md

```diff
@@ -48,21 +48,23 @@ mimir.alerts.kubernetes "<LABEL>" {
-| `bearer_token`           | `secret`            | Bearer token to authenticate with.                                                               |               | no       |
-| `enable_http2`           | `bool`              | Whether HTTP2 is supported for requests.                                                         | `true`        | no       |
-| `follow_redirects`       | `bool`              | Whether redirects returned by the server should be followed.                                     | `true`        | no       |
-| `http_headers`           | `map(list(secret))` | Custom HTTP headers to be sent along with each request. The map key is the header name.          |               | no       |
-| `no_proxy`               | `string`            | Comma-separated list of IP addresses, CIDR notations, and domain names to exclude from proxying. |               | no       |
-| `proxy_connect_header`   | `map(list(secret))` | Specifies headers to send to proxies during CONNECT requests.                                    |               | no       |
-| `proxy_from_environment` | `bool`              | Use the proxy URL indicated by environment variables.                                            | `false`       | no       |
-| `proxy_url`              | `string`            | HTTP proxy to send requests through.                                                             |               | no       |
-| `sync_interval`          | `duration`          | Amount of time between reconciliations with Mimir.                                               | `"5m"`        | no       |
-| `template_files`         | `map(string)`       | A map of Alertmanager [template files][mimir-api-set-alertmgr-cfg].                              |  `{}`         | no       |
+| Name                                  | Type                | Description                                                                                                               | Default         | Required |
+| ------------------------------------- | ------------------- | ------------------------------------------------------------------------------------------------------------------------- | --------------- | -------- |
+| `address`                             | `string`            | URL of the Mimir Alertmanager.                                                                                            |                 | yes      |
+| `global_config`                       | `secret`            | [Alertmanager configuration][global-cfg] to be merged with AlertmanagerConfig CRDs.                                       |                 | yes      |
+| `alertmanagerconfig_matcher_strategy` | `string`            | [AlertmanagerConfigMatcherStrategy][alertmanager-config-matcher-strategy] for adding matchers to AlertmanagerConfig CRDs. | `"OnNamespace"` | no       |
```

Comment:
Below the table we will need a paragraph to list each option for a matcher strategy, along with a short description.